crates/rpc/rpc-engine-api/tests/it/payload.rs

```diff
@@ -42,8 +44,13 @@ fn payload_body_roundtrip() {
                 .map(|x| TransactionSigned::decode(&mut &x[..]))
                 .collect::<Result<Vec<_>, _>>(),
         );
-
-        assert_eq!(block.withdrawals, payload_body.withdrawals);
+        let withdraw = payload_body.withdrawals.as_ref().map(|withdrawals| {
+            withdrawals
+                .iter()
+                .map(|withdrawal| Withdrawal::from(withdrawal.clone())) // Clone if needed, or dereference if they are Copy
+                .collect::<Vec<_>>()
```

Comment:
we don't need this comment here,

clippy --fix should take care of clones that are redundant